app/database/__init__.py

```
@@ -1,4 +1,4 @@
 from .crud_in import *
-from .crud_out import get_user_by_email_and_password
+from .crud_out import get_user_by_email_and_password, get_employee_used_days, get_employee_available_days_for_year, get_employee_total_days_for_year
 
-__all__ = ["create_user", "create_used_vacations", "create_available_vacations", "get_user_by_email_and_password"]
\ No newline at end of file
+__all__ = ["create_user", "create_used_vacations", "create_available_vacations", "get_user_by_email_and_password", "get_employee_used_days", "get_employee_available_days_for_year", "get_employee_total_days_for_year"]
```

Ruff (0.13.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)